### PR TITLE
test: fix the util pipenvInstall() crashing on missing Pipfile.lock

### PR DESCRIPTION
**Pull request by taleinat**
- [ ] Ready for review
- [ ] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

Fixes the test util `pipenvInstall()`.

#### Where should the reviewer start?

`test/test-utils.js`

#### How should this be manually tested?

Run the test suite twice in succession.

#### Any background context you want to provide?

This addresses test failures on PR #47.

### CODE REVIEW

File: test/inspect.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 var test = require('tap').test;
-var fs = require('fs');
+//var fs = require('fs');
```
Comment (michael-go) [inline test/inspect.test.js]: let's remove this line

File: test/test-utils.js [hunk 2]
```diff
@@ -90,24 +90,38 @@ function ensureVirtualenv(venvName) {
...
+    if (proc.status !== 0) {
+      console.error(proc.stdout.toString() + '\n' + proc.stderr.toString());
+      throw new Error('Failed to create virtualenv in ' + venvDir);
     }
-    try {
-      var proc = subProcess.executeSync('virtualenv', [venvDir]);
+    if (process.env.PIP_VER) {
+      proc = subProcess.executeSync(
+        path.resolve(venvDir, binDirName, 'python'),
+        ['-m', 'pip', 'install', `pip==${process.env.PIP_VER}`]
```
Comment (michael-go) [inline test/test-utils.js]: does this install the desired pip inside the virtualenv?

File: test/inspect.test.js (hunk 1 above)
Comment (taleinat) [reply test/inspect.test.js]: It's needed by the test that is temporarily commented out.

File: test/test-utils.js (hunk 2 above)
Comment (taleinat) [reply test/test-utils.js]: Yes, it is calling the virtualenv's python.

(Have to use `python -m pip install pip` to upgrade pip.)

File: test/inspect.test.js (hunk 1 above)
Comment (michael-go) [reply test/inspect.test.js]: yep I know, we can restore it once we un-comment the test - it won't run without it :)

**Pull request merged**